_data/people.yml

```diff
@@ -124,6 +124,12 @@
    email: vatsal.verma@colorado.edu
    pic: vatsal.jpg
 
+- name: Idriss Djiofack Teledjieu
```

Comment:
If you look here https://deploy-preview-214--cranky-jackson-a3e285.netlify.app/people , you'll see the website is not building properly. I believe it's because you need a space before the dash.